### PR TITLE
update

### PR DESCRIPTION
**Pull request by oleksandrduchenko**
leave some comments and suggestions

### CODE REVIEW

File: src/main/java/com/pmt/health/utilities/Jira.java [hunk 1]
```diff
@@ -118,6 +118,7 @@ private static boolean updateIssue(String issue, JsonObject fields) throws IOExc
         String issueId = getIssueId(issue);
         JsonObject json = new JsonObject();
         json.add(FIELDS, fields);
+        System.out.println("HEYYYYYYYYYYYY"+fields.toString());
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/utilities/Jira.java]: did you mean to commit this?

File: src/main/java/com/pmt/health/workflows/UserAdminPage.java [hunk 2]
```diff
@@ -0,0 +1,47 @@
...
+
+    public UserAdminPage(App app, User user) {
+        this.app = app;
+        this.user = user;
+        this.addUserButton = app.newElement(LocatorType.CLASSNAME, "add-user-button");
+        this.loggedInHeading = app.newElement(LocatorType.XPATH, "//h1[text()='User Administration']");
+    }
+
+    public void addUser() {
+        if (addUserButton.is().present()) {
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/workflows/UserAdminPage.java]: is there a scenario in which the addUserButton will not be present?

File: src/test/resources/program.properties [hunk 3]
```diff
@@ -12,4 +12,9 @@ qa.admin.auth.token=TMSHBP7YXKAVDHFG
 staging.url.sub=https://pmtstg.joinallofus.org
 staging.admin.user=matthew.grasberger+pmt@coveros.com
 staging.admin.pass=Password123!
-staging.admin.auth.token=FHKSSFZDQCVPQMLR
\ No newline at end of file
+staging.admin.auth.token=FHKSSFZDQCVPQMLR
+
+kube.url.sub=https://missioncontrol-oduchenko.qak8s.vibrenthealth.com
```
Comment (grasbergerm) [inline src/test/resources/program.properties]: leave this blank, they should be filled in only locally

File: src/main/java/com/pmt/health/utilities/Jira.java (hunk 1 above)
Comment (oleksandrduchenko) [reply src/main/java/com/pmt/health/utilities/Jira.java]: actually yes, cuz this is where I got my track on linking tests to JIra(I can leave more official message if you want to)

File: src/main/java/com/pmt/health/workflows/UserAdminPage.java (hunk 2 above)
Comment (oleksandrduchenko) [reply src/main/java/com/pmt/health/workflows/UserAdminPage.java]: nope, I thought there should be an action to see  where it came from , if there is no button displayed

Comment (grasbergerm) [reply src/main/java/com/pmt/health/workflows/UserAdminPage.java]: no need for the if statement then

File: src/main/java/com/pmt/health/utilities/Jira.java (hunk 1 above)
Comment (grasbergerm) [reply src/main/java/com/pmt/health/utilities/Jira.java]: this shouldn't be added to develop, we don't want print statements like this in the main codebase.

Comment (oleksandrduchenko) [reply src/main/java/com/pmt/health/utilities/Jira.java]: I ll remove it

File: src/main/java/com/pmt/health/workflows/UserAdminPage.java (hunk 2 above)
Comment (oleksandrduchenko) [reply src/main/java/com/pmt/health/workflows/UserAdminPage.java]: alright

File: src/test/resources/program.properties (hunk 3 above)
Comment (oleksandrduchenko) [reply src/test/resources/program.properties]: will be done

File: pom.xml [hunk 4]
```diff
@@ -165,8 +165,17 @@
...
+        <!-- https://mvnrepository.com/artifact/com.github.javafaker/javafaker -->
+        <dependency>
+            <groupId>com.github.javafaker</groupId>
+            <artifactId>javafaker</artifactId>
+            <version>0.16</version>
+        </dependency>
+
     </dependencies>
 
+
```
Comment (grasbergerm) [inline pom.xml]: remove extra lines

File: pom.xml [hunk 5]
```diff
@@ -165,8 +165,17 @@
             <artifactId>websocket-client</artifactId>
             <version>9.4.11.v20180605</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.github.javafaker/javafaker -->
+        <dependency>
+            <groupId>com.github.javafaker</groupId>
```
Comment (grasbergerm) [inline pom.xml]: remove faker

File: src/main/java/com/pmt/health/steps/ui/UserAdminSteps.java [hunk 6]
```diff
@@ -0,0 +1,60 @@
+package com.pmt.health.steps.ui;
+
+import com.github.javafaker.Faker;
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/steps/ui/UserAdminSteps.java]: remove faker

File: src/main/java/com/pmt/health/steps/ui/UserAdminSteps.java [hunk 7]
```diff
@@ -0,0 +1,60 @@
...
+
+public class UserAdminSteps {
+
+    private final User user;
+    private final TestUser testUser;
+    private final DeviceController deviceController;
+    private final UserAdminPage userAdminPage;
+    private final AddUserPage addUserPage;
+
+
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/steps/ui/UserAdminSteps.java]: clean up formatting here

File: src/main/java/com/pmt/health/workflows/UserAdminPage.java [hunk 8]
```diff
@@ -0,0 +1,72 @@
...
+
+    Logger log = Logger.getLogger(LoginPage.class);
+    private User user;
+    private TestUser testUser;
+
+
+    public UserAdminPage(App app, User user, TestUser testUser) {
+        this.app = app;
+        this.user = user;
+
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/workflows/UserAdminPage.java]: clean up formatting in this file too

File: src/main/java/com/pmt/health/workflows/LoginPage.java [hunk 9]
```diff
@@ -127,6 +120,7 @@ public void login(User user) {
         enterMFA(HTTP.obtainOath2Key());
         if (getLoginButton().is().enabled()) {
             getLoginButton().click();
+
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/workflows/LoginPage.java]: delete empty line

File: src/main/java/com/pmt/health/workflows/LoginPage.java [hunk 10]
```diff
@@ -138,10 +132,5 @@ private Element getMFA() {
         return mfaInput;
     }
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/workflows/LoginPage.java]: remove empty lines

File: src/main/java/com/pmt/health/steps/api/AuthSteps.java [hunk 11]
```diff
@@ -15,8 +15,13 @@
     private final UserUtility userUtility;
     private final DeviceController deviceController;
     protected RequestData requestData;
+
     // used to share context of responses and requests between step declarations and workflows
 
+    private String uniqueEmail = "VibrentQA3+" + UserUtility.generateUUID() + "@gmail.com";
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/steps/api/AuthSteps.java]: remove all this, it's not used anywhere

File: src/main/java/com/pmt/health/steps/ui/LoginSteps.java [hunk 12]
```diff
@@ -1,22 +1,28 @@
...
 public class LoginSteps {
 
     private final User user;
+    private final TestUser testUser;
     private final DeviceController deviceController;
     private final LoginPage loginPage;
+    private final UserAdminPage userAdminPage;
 
-    public LoginSteps(DeviceController deviceController, User user) {
+    public LoginSteps(DeviceController deviceController, User user, TestUser testUser) {
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/steps/ui/LoginSteps.java]: no need to pass around the test user, just create it in the step to create a new user

File: src/main/java/com/pmt/health/workflows/AddUserPage.java [hunk 13]
```diff
@@ -0,0 +1,84 @@
...
+    Logger log = Logger.getLogger(LoginPage.class);
+    private User user;
+
+    public AddUserPage(App app, User user) {
+        this.app = app;
+        this.user = user;
+        this.firstNameInput = app.newElement(LocatorType.NAME, "firstName");
+        this.lastNameInput = app.newElement(LocatorType.NAME, "lastName");
+        this.emailInput = app.newElement(LocatorType.NAME, "email");
+        this.rolesInput = app.newElement(LocatorType.XPATH, "//input[contains(@role, 'combobox')]");
```
Comment (grasbergerm) [inline src/main/java/com/pmt/health/workflows/AddUserPage.java]: use css locators for these xpaths

Comment (oleksandrduchenko) [reply src/main/java/com/pmt/health/workflows/AddUserPage.java]: Why do we need to have particularly css?

**Pull request merged**